src/cargo/util/process_builder.rs

```diff
@@ -129,11 +131,11 @@ impl ProcessBuilder {
     }
 }
 
-pub fn process<T: AsRef<OsStr>>(cmd: T) -> CargoResult<ProcessBuilder> {
+pub fn process<T: AsRef<OsStr>, U: AsRef<OsStr>>(cmd: T, cwd: U) -> CargoResult<ProcessBuilder> {
```

Comment:
If it's none then it can just avoid calling `current_dir` on the `Command` that's later used, e.g. it just inherits